portalnet/src/overlay_service.rs

```diff
@@ -2369,6 +2369,19 @@ where
             .collect()
     }
 
+    /// Returns a vector of all the ENRs of nodes currently contained in the routing table which are connected.
+    fn table_entries_enr_connected(&self) -> Vec<Enr> {
```

Comment:
It seems like we could consolidate this method & `table_entries_enr()` with something like
```rust
    fn table_entries_enr(&self, filter_connected: bool) -> Vec<Enr> {
        self.kbuckets
            .write()
            .iter()
            .filter(|entry| {
                // Filter out disconnected nodes.
                if filter_connected {
                    entry.status.is_connected()
                }
            })
            .map(|entry| entry.node.value.enr())
            .collect()
    }
```
well.... I'm not sure if that's the best way to go about this, but if there's an easy way to reduce some of the duplicated lines here, that would be nice to help keep down the size of `overlay_service.rs`